### ISSUE TITLE
Failed to load source: Error Domain=JSServer Code=500 "Unable to resolve module `AccessibilityInfo` from `/Users/

### ISSUE THREAD

**Issue by BeyWood** [facebook/react-native#15357]
Failed to load source: Error Domain=JSServer Code=500 "Unable to resolve module `AccessibilityInfo` from `/Users/

**Comment by hramos** [comment]
Hey, thanks for reporting this issue!

It looks like your description is missing some necessary information, or the list of reproduction steps is not complete. Can you please add all the details specified in the [template](https://github.com/facebook/react-native/blob/master/.github/ISSUE_TEMPLATE.md)? This is necessary for people to be able to understand and reproduce the issue being reported.

I am going to close this, but feel free to open a new issue with the additional information provided. Thanks!

**Comment by BeyWood** [comment]
I find the reason,the reason is  that when i npm install a third component ,some files of  nodemodules been removed!.I must try to re init the reactNative project.
I think its a bug of reactNative

tips:
the component is react-native-picker

**Comment by BeyWood** [comment]
Sorry!My English is not good!
